frontend/src/components/ConsumerGroups/Details/ResetOffsets/ResetOffsets.tsx

```diff
@@ -38,6 +38,7 @@ const ResetOffsets: React.FC = () => {
   return (
     <>
       <PageHeading
```

Comment:
Instead of giving the Pageheading cluaterName couldn't we just use the hook param inside and get it there since we are giving it everywhere

Reply:
@AlexisD02 could you address this?

Reply:
@Mgrdich PTAL